kratos/solving_strategies/strategies/residualbased_newton_raphson_strategy.h

```diff
@@ -1039,6 +1038,10 @@ class ResidualBasedNewtonRaphsonStrategy
         TSystemVectorType& rDx = *mpDx;
         TSystemVectorType& rb  = *mpb;
 
+        if (this->GetEchoLevel() != 0){
+            KRATOS_INFO("NR-Strategy") << "Iteration " << IterationNumber << " / " << mMaxIterationNumber << std::endl;
```

Comment:
I don't agree with this. It fills the simulation with a lot of noise. Many developers already complain about all the noise like @AlejandroCornejo (and myself). I think is smarter to print it during convergence criteria along with the convergence check (I mean on the same line)

Reply:
I see your point and generally agree
Thats why you can disable it, it is protected by the echo-lvl just like the other stuff
It is just as important as the convergence information itself

How would you print from the ConvergenceCriteria, it does not know (and also should not know IMO) abt the max number of iterations=?

Reply:
No, but the maximum number of iterations is not important (at least I don't consider it important). In any case a new variable stored in the ProcessInfo can be created

Reply:
I disagree, I think it is important 
But lets see what others say

Thought this was an easy PR but I guess we have to bother the committee :)

Reply:
I don't see as important. If does not converge a message will be printed. Then you know the maximum number of iterations.

Reply:
I think it is important to know how good your problem converges, how would you plot a convergence-graph without this?

Why is the output noisy? I think with a low echo-lvl there is hardly any output...
Could you please explain what output you don't like? Maybe we can find a global solution for this then

Reply:
You adding a  new line of output per each NL iteration. That's a lot of verbosity

Reply:
If it is important the maximum number of NL can be sabed in a variable in the ProcessInfo